README.md

```diff
@@ -312,6 +312,9 @@ Tips for Fixture Loading Tests
         cache_sqlite_db: true
     ```
 
+    Note: You have to clear the cache every time you modify an entity or a fixture:
```

Comment:
Thanks for you comment. I wanted to cover all the cases but this is too broad. Note that if you use several levels of dependent fixtures, a fixture is not necessarily referenced by the main fixture.

E.g.: A depends on B, B depends on C . If you modify C, LiipFunctionalTestBundle won't detect the change.